lib/api.js

```diff
@@ -83,8 +83,8 @@ class RPC {
   start(configPath, callback, unsafeFlag=false) {
     let config = null;
 
-    if (this.shares.size + 1 >= cpus().length && !unsafeFlag) {
-      return callback(new Error('insufficient system resources available'));
+    if (this.shares.size >= cpus().length && !unsafeFlag) {
```

Comment:
On a single core system this will still throw an error
1 farmer >= 1 core
it should be
1 farmer > 1 core